using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -169,287 +107,104 @@ Campaign.resetLinkageFields()
 ```swift
 ACPCampaign.resetLinkageFields()
 ```
+
 {% endtab %}
 
 {% tab title="React Native" %}
+
 #### JavaScript
 
-### resetLinkageFields
+#### Syntax
+
+```javascript
+resetLinkageFields();
+```
+
+#### Example
 
 ```javascript
 ACPCampaign.resetLinkageFields();
 ```
+
 {% endtab %}
 {% endtabs %}
 
-## Set up push messaging
+## setLinkageFields
 
-To enable push messaging with Adobe Campaign, the push identifier that is received from the Apple Push Notification Service \(APNS\) or Firebase Cloud Messaging \(FCM\) must be sent to the Adobe Identity service by calling `setPushIdentifer`. After the API is invoked, a network request is made to Campaign that contains the message interaction event. For more information about Campaign message interaction events, see [Implementing local notification tracking](https://helpx.adobe.com/campaign/kb/local-notification-tracking.html#Description).
+This API sets the Campaign linkage fields \(CRM IDs\) in the Mobile SDK that are used to download personalized messages from Campaign. The set linkage fields are stored as a base64-encoded JSON string in memory, and they are sent in a custom HTTP header `X-InApp-Auth` in all future Campaign rules download requests until `resetLinkageFields` is invoked. These in-memory variables are lost in the following scenarios:
 
-For more information about setting up your iOS app to connect to APNS and retrieve a device token that will be used as a push identifier, see [Registering Your App with APNs](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns?language=objc). For more information about setting up your Android app to connect to FCM and retrieve a device registration token that will be used as a push identifier, see [Set up a Firebase Cloud Messaging client app on Android](https://firebase.google.com/docs/cloud-messaging/android/client).
+- When an application crash event occurs, after a graceful termination of the application.
```

Comment:
This sentence doesn't make sense. These are two different cases - 1) app crash and 2) graceful app termination